elodie/external/pyexiftool.py

```diff
@@ -151,8 +151,16 @@ def format_error (result):
         else:
             return 'exiftool finished with error: "%s"' % strip_nl(result) 
 
+class Singleton(type):
+    """Metaclass to use the singleton [anti-]pattern"""
+    instance = None
 
-class ExifTool(object):
+    def __call__(cls, *args, **kwargs):
+        if cls.instance is None:
+            cls.instance = super(Singleton, cls).__call__(*args, **kwargs)
+        return cls.instance
+
+class ExifTool(object, metaclass=Singleton):
```

Comment:
This line causes an [error](https://travis-ci.org/jmathai/elodie/jobs/636196855#L1648) in the python2 unit tests.

```
SyntaxError (invalid syntax (pyexiftool.py, line 163)) ... ERROR
```

Reply:
Singleton is now Python 2 & 3 compatible.